### ISSUE TITLE
Osmotic diarrhea

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6400]
**Preferred term label:**
Osmotic diarrhea

**Synonyms**
Osmotic diarrhoea

**Definition (free text, please give PubMed ID)**
Watery stool characterized by an unaccounted gap between the stool water electrolytes and the measured osmolality.
PMID: 27773805

**Parent term (use hpo.jax.org/app)**
Diarrhea

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: # 618168

**Comment by pnrobinson** [comment]
adding term, thanks
